auto-value-gson/src/test/java/com/ryanharter/auto/value/gson/AutoValueGsonExtensionTest.java

```diff
@@ -741,7 +743,7 @@ public void simpleNoFieldNamePolicy() {
         + ".getType()).getType()).getType()).getType()));\n"
         + "        }\n"
         + "        "
-        + "map__string_map__string_map__string_map__string_map__string_wildcard__string_adapter"
```

Comment:
nit: looks like the formatting got messed up in this file.

Reply:
I'm not sure really what to do about that, I just regenerated this one and intelliJ wants to line chomp it now :|

Reply:
I agree, preserve the indenting in spite of intelliJ.